src/components/cc-input-text/cc-input-text.js

```diff
@@ -255,7 +234,8 @@ export class CcInputText extends LitElement {
           ` : ''}
         </label>
       ` : ''}
-
+      
+      
```

Comment:
**🤓 nitpick:** Those two empty lines has been committed mistakenly I guess (non blocking).

Reply:
Thanks, I have left one empty line as it was originally to espace the code :)